runtime/onert/backend/train/Tensor.h

```diff
@@ -43,7 +44,23 @@ class Tensor : public basic::Tensor
   bool applyShape(const ir::Shape &) override { return false; }
 };
 
-using ExternalTensor = basic::ExternalTensor;
+class TrainableTensor : public Tensor, public train::ITrainableTensor
+{
+public:
+  TrainableTensor() = delete;
+
+public:
+  TrainableTensor(const ir::OperandInfo &info, const ir::Layout layout) : Tensor{info, layout}
+  {
+    // DO NOTHING
+  }
+
+public:
+  void applyGradient(const IPortableTensor &grad_tensor, double lr) override;
+
+public:
+  void fillBuffer(const std::shared_ptr<ir::Data> &data);
```

Comment:
@ragmani `fillBuffer` is not a part of interface `ITrainableTensor`. Is `fillBuffer` a part of `TrainableTensor` only?

Reply:
Yes, there will be filled in `train` backend. So, I think `ITrainableTensor` does not need to have `fillBuffer` now.

Reply:
I wondered the similar method would exists in `Tensor`. But it has only `setBuffer(uint8_t buf)`.